src/js/components/Anchor/doc.js

```diff
@@ -2,6 +2,52 @@ import { describe, PropTypes } from 'react-desc';
 
 import { genericProps, getAvailableAtBadge } from '../../utils';
 
+export const themeDoc = {
+  'global.focus.border.color': {
+    description: 'The color around the Anchor when in focus.',
+    type: 'string | { dark: string, light: string }',
+    defaultValue: '#FD6FFF',
+  },
+  'anchor.color': {
+    description: 'Used to apply a default color to the icon and label.',
+    type: 'string | { dark: string, light: string }',
+    defaultValue: "{ light: '#1D67E3', dark: '#6194EB' }",
+  },
+  'anchor.fontWeight': {
+    description: 'The font weight to be applied to the anchor text.',
```

Comment:
"to be applied" seems a bit heavy. Also, we should be more consistent about our use of the terms "text" and "label". How about just "The font weight of the label"?